packages/wallet-sdk/src/connection/ServerMessage.ts

```diff
@@ -76,3 +76,14 @@ export interface ServerMessageEvent extends ServerMessage {
   event: string;
   data: string;
 }
+
+export function isServerMessageEvent(msg: any): msg is ServerMessageEvent {
+  return (
+    msg &&
+    msg.type === 'Event' &&
+    typeof msg.sessionId === 'string' &&
+    typeof msg.eventId === 'string' &&
+    typeof msg.event === 'string' &&
+    typeof msg.data === 'string'
+  );
+}
```

Comment:
following the existing pattern. will revisit later.
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/302a8f55-adfb-419e-a6e4-de4509ad54ce)